components/automate-chef-io/content/docs/eas.md

```diff
@@ -0,0 +1,87 @@
+    weight = 10
++++
+
+The Chef Enterprise Application Stack (EAS) allows organizations to automate infrastructure, security, and application delivery together, helping them deploy software quickly while maintaining compliance with industry regulations. Chef EAS helps teams drive efficiency and consistency for any application across multi-cloud and heterogeneous infrastructure.
+
+Chef EAS includes the following Chef solutions:
+
+* **Chef Habitat** builds and deploys your applications because delivering applications is where you derive your business value
+* **Chef Infra** configures your infrastructure, because your applications also needs to run on a stable and scalable system
+* **Chef Inspec** validates and secures both your applications and your infrastructure
+* **Chef Automate** visualizes your applications, their infrastructure, and the compliance status for both.
+* **Chef Workstation** provides a unified experience for all of Chef EAS
+
+## Chef EAS Application Management with Chef Habitat
+
+Chef Habitat automates building and deploying applications, turning a web of delicate procedures into a resilient and repeatable process.
+A Chef Habitat package contains the compiled binary and a manifest of all the dependencies required for building and running your application.
+This results in a lightweight and portable artifact that you can deploy to bare metal or virtual machines, as well as export to immutable formats, such as Docker containers or Kubernetes pods.
+Bundling applications with a complete dependency manifest ensures that your application behaves as consistently on a developer's laptop as it does in its on-prem VM farm staging environment, and it also behaves as consistently there as it does in its cloud-based deployment environment.
+
+## The Chef Automate EAS Application Tab
+
+With the new application operations dashboard in Chef Automate, operations teams gain comprehensive and customizable visibility into the health of the services that make up the application. Making it easier to understand what’s degrading the health of an applications and to keep it running smoothly.
+
+The Chef Automate EAS Application dashboard allows you to organize and display your applications data from Chef Habitat in an intuitive way. The Chef Automate EAS Application Dashboard provides visibility into your application artifacts and channels by letting you see which versions of your packages are running on your system.
+Chef Automate organizes data from the application and environment tags provided by the Chef Habitat supervisor.
+
+## Prerequisites
+
+The Chef Automate EAS application feature introduces several concepts from Chef Habitat, which are introduced in the following [Glossary]({{< relref "applications.md#Glossary" >}}).
+
+Find Chef Automate EAS system requirements, installation, and configuration instructions in the [Setting up the Applications Dashboard]({{< relref "applications-setup.md" >}}).
+
+### Glossary
+
+_Application_
```

Comment:
Do we want to put a dash or something to separate the term from the definition? The site just puts the two lines together as in a paragraph otherwise.